WEB_SERVICE/DNS/_MEDIUM/DNS_TXT_RECORDS_MALICIOUS_CONTENT/meta.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "DNS Vulnerability: Malicious Content in TXT Records",
+  "short_description": "A vulnerability where DNS TXT records are used for malicious purposes such as data exfiltration, malware command execution, or DNS tunneling.",
+  "risk_rating": "medium",
+  "references": {
+    "Splunk Deep Learning Blog": "https://www.splunk.com/en_us/blog/security/ml-in-security-detect-suspicious-txt-records-using-deep-learning.html",
+    "AhnLab Security Blog": "https://asec.ahnlab.com/en/54916/",
+    "ProSec Networks": "https://www.prosec-networks.com/en/blog/dns-tunneling-erkennen/"
+  },
+  "privacy_issue": true,
+  "security_issue": true,
+  "categories": {
+    "SOC2_CONTROLS": [
```

Comment:
I don't think `CC_7_4` is relevant because rollback and recovery processes focus on operational continuity rather than directly on preventing or detecting malicious DNS activity, unauthorized software, or data exfiltration.